packages/kbn-es-query/src/filters/lib/exists_filter.ts

```diff
@@ -31,3 +31,7 @@ export type ExistsFilter = Filter & {
 };
 
 export const isExistsFilter = (filter: any): filter is ExistsFilter => filter && filter.exists;
```

Comment:
`isExistsFilter` and the other filter type checking function names are a bit of a misnomer. One would expect a boolean return from a method with a name pre-fixed with `is`. Should we consider changing the function names or their signature to something similar to what you've done in [`isMatchPhraseFilter`](https://github.com/elastic/kibana/pull/49537/commits/002cfe3d4fe8f85dcd14e5056d3aed08f67c409c#diff-e7ba505d30dc7279d78fb5f46fd53803R49) , that now returns a boolean? It doesn't have to be in this PR.

Reply:
The names make sense because these are meant to be [custom type guards](https://www.typescriptlang.org/docs/handbook/advanced-types.html#using-type-predicates). You're right though, it wouldn't hurt to cast the return value to a boolean. Could be a fix it week issue.